scripts/tools/annotation/templates/up_liste.html

```diff
@@ -10,52 +10,46 @@ <h2>Video: {{video_name}}</h2>
     <input type="hidden" id="num" name="num" value="{{num}}">
     <input type="hidden" id="n_images" name="n_images" value="{{n_images}}">
     {% if images %}
-        <ul class="fichiers">
-        {% for imgs in images %}
-           <div class="row">
-            <table>
-                <tr>
+        <ul class="fichiers" style="padding: 0px;" >
+            {% for imgs in images %}
+            <div class="row" style="display: flex;">
                 {% for img, num, pred_class in imgs %}
-                    <td>
+                <div class="col" style="margin:10px">
```

Comment:
I think we have css files. can we use that ?

Reply:
yes, it was just for the testing purpose, now I'll clean the code for final push.

Reply:
Thank you though